### PR TITLE
Create clean ind cqc filled posts script

### PR DESCRIPTION
**Pull request by james-francis-MT**
# Description
Initial implementation of clean IND cqc filled posts script 

# How to test
Has been tested in branch and passes but data, partition keys etc may not be correct yet

https://trello.com/c/fufgEGCv/207-epic-8-refactor-job-name-and-terraform-must

# Developer checklist
- [x] Unit test
- [x] Linked to Trello ticket
- [x] Documentation up to date

### CODE REVIEW

File: tests/test_file_generator.py [hunk 1]
```diff
@@ -919,6 +920,61 @@ def generate_prepared_locations_preclean_file_parquet(
     return df
 
 
+def generate_ind_cqc_filled_posts_file_parquet(partitions=["2022", "03", "08"]):
```
Comment (james-francis-MT) [inline tests/test_file_generator.py:923]: Created a clone of the data generator for cleaned_prepare_loc script. This should be moved to test data and test schema files in the refactor ticket

File: jobs/clean_ind_cqc_filled_posts.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    )
+
+    locations_df = remove_unwanted_data(locations_df)
+
+    locations_df = replace_zero_beds_with_null(locations_df)
+    locations_df = populate_missing_carehome_number_of_beds(locations_df)
+
+    locations_df = null_job_count_outliers(locations_df)
+
+    locations_df = create_partition_keys_based_on_todays_date(locations_df)
```
Comment (james-francis-MT) [inline jobs/clean_ind_cqc_filled_posts.py:52]: Get rid

File: jobs/clean_ind_cqc_filled_posts.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+    print(f"Exporting as parquet to {cqc_filled_posts_cleaned_destination}")
+
+    utils.write_to_parquet(
+        locations_df,
+        cqc_filled_posts_cleaned_destination,
+        mode="append",
+        partitionKeys=["run_year", "run_month", "run_day", "import_date"],
```
Comment (james-francis-MT) [inline jobs/clean_ind_cqc_filled_posts.py:60]: Switch to standard

File: jobs/clean_ind_cqc_filled_posts.py [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    cqc_filled_posts_source: str,
+    cqc_filled_posts_cleaned_destination: str,
+) -> pyspark.sql.DataFrame:
+    print("Cleaning cqc_filled_posts dataset...")
+
+    locations_df = utils.read_from_parquet(cqc_filled_posts_source).selectExpr(
+        *COLUMNS_TO_IMPORT
+    )
+
+    locations_df = remove_unwanted_data(locations_df)
```
Comment (james-francis-MT) [inline jobs/clean_ind_cqc_filled_posts.py:45]: No need for this

**Review by MarionJHolloway — APPROVED** [review]
Looks good!

**Pull request merged**